### DISCUSSION TITLE
Error message "Something went wrong."

### DISCUSSION THREAD

**Discussion by MagicLike** [umami-software/umami#1211] [Q&A] [answered]
Like in #503 & #495 I sometimes get the Error message "Something went wrong.", while browsing through the Umami Dashboard & the Details page of a website. A few reloads with the button next to the time period selection fix the error messages, but not always.
![grafik](https://user-images.githubusercontent.com/82117109/162613568-2bbecb32-a449-49a1-9d61-7aab69b5c642.png)
![grafik](https://user-images.githubusercontent.com/82117109/162613571-e8d4b8b1-2e5d-44bc-9b14-e65e28171f8c.png)

**Comment by mikecao** [comment]
Check the network tab of the developer tools to see what error is thrown.

**Comment by MagicLike** [comment]
I see these 500 errors
![grafik](https://user-images.githubusercontent.com/82117109/166119148-a952e8c3-2c4a-46c3-8181-cfebe5221a16.png)

**Comment by mikecao** [comment]
Where are you hosting? You'll have to check the logs. If hosting locally, you can run in dev mode and look at the logs.

**Comment by MagicLike** [comment, marked as answer]
I think I figured it out:
I am hosting Umami on Vercel and the database external.
Apparently the database has too many connections, but is only accessed by one program: 

> Error querying the database: Server error: `ERROR 42000 (1203): User ******** already has more than 'max_user_connections' active connections'

![grafik](https://user-images.githubusercontent.com/82117109/166140039-5e6b0276-30c7-4a3a-8305-ca1c90e5b68e.png)

Is it possible, that Umami creates a new connection per request, or something like that?

**Reply by MagicLike** [reply]
@mikecao Do you know if Umami establishes a new connection per request or are all requests to the database handled via one connection?
If there is only one connection, I can't explain the error.
